### PR TITLE
Clear Tasks of Persons

### PR DESCRIPTION
**Pull request by chin-herng**
# The `cleartask` Command

The `cleartask` command unassigns all tasks to people in the address book. The command has the following format:

```
cleartask PERSON_NUMBER
```

Parameter: `PERSON_NUMBER`

- Must be a numeric string representing a number between 1 and number of persons in the filtered person view.
- If the value is not acceptable, the error message The person index provided is invalid is displayed.

This PR is linked to #43.

### CODE REVIEW

**Review by nobodyishappy — APPROVED** [review]
Looks good to me!

**Review by Yskie — APPROVED** [review]
LGTM!

**Review by breezetall — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/ClearTaskCommand.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            + "Parameters: PERSON_INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_SUCCESS = "Tasks for %1$s has been cleared.";
+
+    private final Index index;
+
+    /**
+     * @param index of the person in the filtered person list to edit the remark
+     */
```
Comment (Kaya3842) [inline src/main/java/seedu/address/logic/commands/ClearTaskCommand.java:34]: Should this be "Index of the person..." instead of "of the person..." ?

**Review by Kaya3842 — APPROVED** [review]
Looks good to me!

**Pull request merged**